docs/resources/resource.md

```diff
@@ -55,13 +64,24 @@ resource "twingate_resource" "resource" {
 
 ### Optional
 
+- `access` (Block List, Max: 1) Restrict access to certain groups or service accounts (see [below for nested schema](#nestedblock--access))
 - `group_ids` (Set of String, Deprecated) List of Group IDs that have permission to access the Resource, cannot be generated by Terraform and must be retrieved from the Twingate Admin Console or API
 - `protocols` (Block List, Max: 1) Restrict access to certain protocols and ports. By default or when this argument is not defined, there is no restriction, and all protocols and ports are allowed. (see [below for nested schema](#nestedblock--protocols))
 
 ### Read-Only
 
 - `id` (String) Autogenerated ID of the Resource, encoded in base64
 
+<a id="nestedblock--access"></a>
+### Nested Schema for `access`
+
+Optional:
+
+- `group_ids` (Set of String) List of Group IDs that have permission to access the Resource.
+- `non_authoritative` (Boolean) Determines authoritative behaviour for handling the Resource's groups or service accounts.
```

Comment:
@vmanilo why is the property called `non_authoritative` ? marking it false is double negative - not not authoritative?  it should just be straightforward `authoritative`

Reply:
yes, we want to have `authoritative` mode by default, means it should be `true` by default even when user not set it manually, in that case terraform will show drift because in the state will be added  authoritative=true